modules/core/src/main/java/org/apache/ignite/spi/metric/jmx/ReadOnlyDynamicMBean.java

```diff
@@ -57,16 +58,18 @@ else if ("invoke".equals(actionName))
 
     /** {@inheritDoc} */
     @Override public AttributeList getAttributes(String[] attributes) {
-        AttributeList list = new AttributeList();
-
+        AttributeList attrList = new AttributeList();
+        List<Attribute> list = attrList.asList();
         try {
-            for (String attr : attributes) {
-                Object val = getAttribute(attr);
-
-                list.add(val);
+            for (String attribute : attributes) {
+                Object val = getAttribute(attribute);
```

Comment:
Let's keep the line break

Reply:
The line break violates the Ignite coding guide lines. Under "whitespace and empty lines" it says

"Note that single blank line SHOULD NOT be used for the mere decoration of the source code."

Reply:
See table above. The section `Between minimal semantic units in the method or initializer body` says to use 1 line

Reply:
Oh wow - that makes for a lot of blank lines! Well if that is the way it is done here :)